### ISSUE TITLE
have you check crawled urls?

### ISSUE THREAD

**Issue by weaming** [kabelsea/go-scrapy#21]
I did not find the logic in code

**Comment by kabelsea** [comment]
No, library dont check crawled url, but I think to implement this functionality in future)

**Comment by weaming** [comment]
So I maybe have crawled the same url more than one time? I thinks it's a fundamental feature all the time..
